test/node-api/test_general/test_general.c

```diff
@@ -26,6 +26,57 @@ static napi_value testGetNodeVersion(napi_env env, napi_callback_info info) {
   return result;
 }
 
+static napi_value testBufferFromArrayBuffer(napi_env env,
+                                            napi_callback_info info) {
+  napi_status status;
+  napi_value arraybuffer;
+  void* data;
+  napi_value buffer;
+  size_t byte_length = 1024;
+  size_t byte_offset = 0;
+
+  status = napi_create_arraybuffer(env, byte_length, &data, &arraybuffer);
+  NODE_API_ASSERT(env, status == napi_ok, "Failed to create arraybuffer");
+
+  status = node_api_create_buffer_from_arraybuffer(
+      env, arraybuffer, byte_length, byte_offset, &buffer);
```

Comment:
The position of `byte_offset` and `byte_length` doesn't match the signature.

Reply:
thanks